lib/winapi/filemanip.c

```diff
@@ -241,6 +241,138 @@ BOOL MoveFileA (LPCTSTR lpExistingFileName, LPCTSTR lpNewFileName)
     }
 }
 
+BOOL CopyFileA (LPCTSTR lpExistingFileName, LPCTSTR lpNewFileName, BOOL bFailIfExists)
+{
+    NTSTATUS status;
+    HANDLE sourceHandle;
+    HANDLE targetHandle = INVALID_HANDLE_VALUE;
+    ANSI_STRING sourcePath;
+    ANSI_STRING targetPath;
+    OBJECT_ATTRIBUTES objectAttributes;
+    IO_STATUS_BLOCK ioStatusBlock;
+    FILE_NETWORK_OPEN_INFORMATION networkOpenInformation;
+    LPVOID readBuffer = NULL;
+    SIZE_T readBufferRegionSize = 0x1000;
+    DWORD bytesRead;
+
+    RtlInitAnsiString(&sourcePath, lpExistingFileName);
+    InitializeObjectAttributes(&objectAttributes, &sourcePath, OBJ_CASE_INSENSITIVE, ObDosDevicesDirectory(), NULL);
+
+    status = NtOpenFile(
```

Comment:
I suggest to replace this with a call to `CreateFile` to reduce duplicated code.